gsy_framework/sim_results/cumulative_grid_trades.py

```diff
@@ -192,10 +192,12 @@ def _update_area_children_in_accumulated_trades_dict(self, accumulated_trades, a
         Returns: None
         """
         current_child_uuids = [
-            c["uuid"] for c in accumulated_trades[area["uuid"]]["children"]
+            child["uuid"] for child in accumulated_trades[area["uuid"]].get("children", []) or []
```

Comment:
nit, this should be enough to cover the non existing key and the case when it exists and equals to None
```suggestion
            child["uuid"] for child in accumulated_trades[area["uuid"]].get("children") or []
```